PyRDF/Proxy.py

```diff
@@ -10,20 +10,25 @@ class Proxy(object):
 
     """
```

Comment:
I would reword the docstring of this class:
"""
Instances of Proxy act as futures of the result produced by some action.
They implement a lazy synchronization mechanism: when accessed for the first time, they trigger the execution of the whole RDataFrame graph.
"""